integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -427,9 +429,20 @@ public void testDomainRestart() {
     result = executeLifecycleScript(STOP_DOMAIN_SCRIPT, DOMAIN, null);
     assertTrue(result.contains("No changes needed"), "stopDomain.sh shouldn't make changes");
 
+    // verify managed server instances can not be started while 
+    // spec.serverStartPolicy is NEVER 
```

Comment:
I assume domain's spec.serverStartPolicy  is NEVER. Probably comment can say that "managed server instances can't be started while domain is stopped".

Reply:
Modified the comments.

Reply:
update the use case description for the changes